addons/default/visiosoft/advs-module/resources/views/list/partials/list-filter.twig

```diff
@@ -1,20 +1,19 @@
 {% set formHtml %}
     <form action="{{ url_route('visiosoft.module.advs::list') }}" method="get">
 {% endset %}
-{% if 'cat' in param|keys %}
-    {% if not param['cat'] is empty %}
-        {% set catId = entries('cats', 'category').find(param['cat']) %}
-        {% set cityId = null %}
-        {% if 'city' in param|keys %}
-            {% set citiesArray = param['city'][0]|split(',') %}
-            {% if count(citiesArray) is same as(1) %}
-                {% set cityId = entries('location', 'cities').find(param['city'][0]) %}
-            {% endif %}
-        {% endif %}
-        {% set formHtml %}
-            <form action="{{ url_route('adv_list_seo', [catId.slug, cityId.slug]) }}" method="get">
-        {% endset %}
-    {% endif %}
+{% set catSlug = null %}
+{% set citySlug = null %}
+{% set pathInfo = app.request.pathinfo|split('/') %}
+{% if pathInfo[1] == 'c' %}
+    {% set catSlug = pathInfo[2] %}
+{% endif %}
+{% if pathInfo|length is same as(4) %}
```

Comment:
Why do you check the path info for the second time for citySlug ?

Reply:
the first time is to check which path are we using. the second time is to check if city is in the path that we are using @ozcan-durak